docs/docs/references/cli/cli.md

```diff
@@ -43,7 +43,7 @@ to initialize an empty project.
 
```

Comment:
Move this file to `references/cli-cheat-sheet`

Reply:
A separate dir or .md file?

Reply:
The name of this file associates with the folder and folder takes the name from the file `TITLE`. If I rename it, then the folder name will change to lowercase `cli`.

Reply:
See https://docusaurus.io/docs/sidebar/autogenerated#category-index-convention

Reply:
changing to this
![image](https://user-images.githubusercontent.com/14215045/235686180-7e57e18d-2dd9-413f-a32e-b76e1e1a1800.png)